packages/config/config/devices/0x019b/z-trm6.json

```
@@ -0,0 +1,469 @@
+{
+	"manufacturer": "Heatit",
+	"manufacturerId": "0x019b",
+	"label": "Z-TRM6",
+	"description": "Floor thermostat",
+	"devices": [
+		{
+			"productType": "0x0030",
+			"productId": "0x3001",
+			"zwaveAllianceId": 4908
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Binary Switch Set",
+			"maxNodes": 10
+		},
+		"3": {
+			"label": "Thermostat Setpoint Set",
+			"maxNodes": 10
+		},
+		"4": {
+			"label": "Thermostat Mode Set",
+			"maxNodes": 10
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Disable Buttons",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Enable",
+					"value": 0
+				},
+				{
+					"label": "Disable",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Sensor Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 1,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Floor",
+					"value": 0
+				},
+				{
+					"label": "Internal",
+					"value": 1
+				},
+				{
+					"label": "Internal with floor limit",
+					"value": 2
+				},
+				{
+					"label": "External",
+					"value": 3
+				},
+				{
+					"label": "External with floor limit",
+					"value": 4
+				},
+				{
+					"label": "Power regulator",
+					"value": 5
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "Sensor Value(sen)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 0,
+			"unsigned": true,
+			"unit": "KΩ",
+			"options": [
+				{
+					"label": "10",
+					"value": 0
+				},
+				{
+					"label": "12",
+					"value": 1
+				},
+				{
+					"label": "15",
+					"value": 2
+				},
+				{
+					"label": "22",
+					"value": 3
+				},
+				{
+					"label": "33",
+					"value": 4
+				},
+				{
+					"label": "47",
+					"value": 5
+				},
+				{
+					"label": "6,8",
```

Is this supposed to be a comma between 6 and 8? Seems like a strange sequence of numbers in that case.

---

Ok seems to be intended. Weird... We should turn it into a decimal point for consistency though.
```suggestion
					"label": "6.8",
```